roles/remediate/tasks/main.yml

```
@@ -0,0 +1,16 @@
+---
+# tasks file for remediations
+
+- name: Check if the system is RHEL 8
+  ansible.builtin.assert:
+    that: ansible_distribution == 'RedHat' and ansible_distribution_major_version|int == 8
+    msg: "This role is only applicable to RHEL 8"
+  tags: remediate
+
+- name: Remediate the system
+  ansible.builtin.include_tasks: "{{ item }}.yml"
```

I tried testing the role and it failed here with 'item' is undefined error... 

```suggestion
  ansible.builtin.include_tasks: "{{ remediation }}.yml"
```